packages/gatsby-theme-docs/src/components/image.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { useResizedImagesByPath } from '../hooks/use-resized-images';
+
+const Img = styled.img`
+  display: block;
+  background-color: #fff;
+  margin: auto;
+`;
+
+const Image = (props) => {
+  const resized = useResizedImagesByPath(props.src);
+  return resized ? (
+    <figure className="gatsby-resp-image-figure">
+      <a href={resized.originalImg} target="ct-docs-original-size-image">
```

Comment:
What does the `ct-docs-original-size-image` target do?

Reply:
Let's use `target="_blank"`

```suggestion
      <a href={resized.originalImg} target="_blank">
```